docs/src/layouts/Main.astro

```diff
@@ -23,7 +23,7 @@ if (currentPage) {
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     
     <title>{content.title ? `${content.title} 🚀 ${site.title}` : site.title}</title>
-    <MetaData {content} canonicalURL={Astro.request.canonicalURL}/>
+    <MetaData {content} {site} canonicalURL={Astro.request.canonicalURL}/>
```

Comment:
I missed this in the original MetaData PR: do we want to support `{content}` vs. `content={content}`? I wasn't aware that this worked!

Fine to experiment with this for now in our own site, but whenever we upstream this back to the docs template I don't think if we're ready to make this syntax "blessed" yet.

Reply:
In the sense of saying "this is the code pattern you should use"? I do like the aspect where it feels similar to the spread API in javascript where you can add a variable to an object without writing something like
```js
const x = {
  "html": html
};
```

Reply:
yea I like that it saves characters, and if it were JSX i would love it, but it feels a bit "not HTML" and we do claim to be a superset of HTML... not too worried about it but just surprised that this works!